src/bundle/Resources/translations/forms.en.xliff

```diff
@@ -491,21 +491,6 @@
         <target>User / Selfedit</target>
         <note>key: role.policy.user.selfedit</note>
       </trans-unit>
-      <trans-unit id="30c5e1c11a520e499ad68130a2c7b844c2243888" resname="role.policy.workflow">
```

Comment:
Even though this should not be a part of AdminUI's translations I wouldn't remove it in this PR because it will require moving these translation strings to `ezsystems/ezplatform-workflow`.